lib/asciidoctor/substitutors.rb

```diff
@@ -131,7 +131,7 @@ def apply_subs source, subs = NORMAL_SUBS, expand = nil
       when :post_replacements
         text = sub_post_replacements text
       else
-        warn %(asciidoctor: WARNING: unknown substitution type #{type})
+        logger.warn %(unknown substitution type #{type})
```

Comment:
Agreed, the next step in the evolution of the logger would be to add categories. But before we do that, I think we need to decide whether these messages should actually be document messages instead of logger messages. That's where a category might fit best. See #2657.